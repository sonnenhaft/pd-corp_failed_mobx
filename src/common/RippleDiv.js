import React from 'react'

import Ripple from 'react-toolbox/lib/ripple'

// eslint-disable-next-line no-unused-vars
export const RippleDiv = Ripple({ spread: 1 })(({ theme, children, ...props }) => {
  return <div { ...props } style={ { position: 'relative' } }>
    {children}
  </div>
})
