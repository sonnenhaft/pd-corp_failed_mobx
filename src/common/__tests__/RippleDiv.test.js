import React from 'react'
import { RippleDiv } from '../RippleDiv'
import renderer from 'react-test-renderer'

test('RippleDiv code should now changed', () => {
  const component = renderer.create(
    <RippleDiv page="http://www.facebook.com">Facebook</RippleDiv>
  )
  const tree = component.toJSON()
  expect(tree).toMatchSnapshot()
})
