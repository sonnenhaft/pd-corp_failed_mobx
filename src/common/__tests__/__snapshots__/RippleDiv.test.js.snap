// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`RippleDiv code should now changed 1`] = `
<div
  disabled={false}
  onMouseDown={[Function]}
  onTouchStart={[Function]}
  page="http://www.facebook.com"
  style={
    Object {
      "position": "relative",
    }
  }
>
  Facebook
</div>
`;
